binder/src/main/java/io/grpc/binder/BinderChannelBuilder.java

```diff
@@ -114,24 +173,28 @@ public static BinderChannelBuilder forTarget(String target) {
   }
 
   private final ManagedChannelImplBuilder managedChannelImplBuilder;
+  private final BinderChannelCredentials channelCredentials;
```

Comment:
Do we need this instance field? Looks like it's not used.

Reply:
Removed.